### ISSUE TITLE
Test with a title as a template string is always executed

### ISSUE THREAD

**Issue by gfomichev** [annaet/cypress-tags#16]
If your test title is a template string (enclosed by the backtick), it is not processed by tag filtering.
Can be reproduced in _cypress-tags-example_. Replace single quotes in the _I am a regular test_ test title with backticks (`)
```
describe('Run tests with no tags', () => {
  it(`I am a regular test`, () => {});
});
```
_I am a regular test_ test will be executed even if only CYPRESS_INCLUDE_TAGS config is set.

**Comment by annaet** [comment]
@gfomichev thanks for this report, I'll look into fixing this!

**Comment by annaet** [comment]
@gfomichev this was actually fixed in version `0.0.19`, but I hadn't updated the version in `cypress-tags-example` so thanks for pointing that out!

If you pull down the latest code, I've updated the version and added some more example tests. I'll close this issue for now, but please let me know if you have anymore problems.

**Comment by gfomichev** [comment]
@annaet I'm still able to reproduce the original behavior. 
If your template string doesn't contain any expression (just a string enclosed by the backtick) the test is executed.

In file `integration\template_expressions.spec.ts` remove `${foo}` from line 4:
```
...
describe(`Run tests with no tags ${foo}`, () => {
  it(`I am a regular test`, () => { });             <== HERE
  it('smoke', `I am a smoke test ${foo}`, () => { });
  it('wip', `I am a wip test ${foo}`, () => { });
});
...
```
This `I am a regular test` test will be executed even if you set CYPRESS_INCLUDE_TAGS.

Could you check that, please?

**Comment by annaet** [comment]
Ah yes you're right! I've been able to recreate this, will take a look later today.

**Comment by annaet** [comment]
Hi @gfomichev - this should be fixed now in `v0.0.20`.
